dev/doc/MERGING.md

```diff
@@ -37,6 +37,9 @@ When maintainers receive a review request, they are expected to:
   REVIEWERS(*) have approved the PR, the assignee is expected to follow the merging
   process described below.
 
+To know what files you are a code owner of in a large PR, you can run
+`dev/tools/check-owners-pr.sh xxxx`.
```

Comment:
Maybe add a note that it can produce incorrect results.

Reply:
Also you may give a hint of what we can expect from running this tool: a list of files? a smaller diff?

Reply:
>Maybe add a note that it can produce incorrect results.

"Results are unfortunately imperfect." or something more specific.

>Also you may give a hint of what we can expect from running this tool: a list of files? a smaller diff?

It produces a list of files, isn't it clear from the current formulation?